CONTRIBUTING.md

```diff
@@ -1,6 +1,24 @@
 # Contributing
 
+## Adding plugins
+
+When adding or upgrading a plugin, we should see changes in both `package.json`
+and `yarn.lock`.
+
+You'll need to [install yarn](https://yarnpkg.com/en/docs/install).
```

Comment:
> For me, running yarn outside of the container is a better experience. What are the reasons you'd prefer not to?

For the same reason I don't run `bundle` locally to update `Gemfile.lock`, and for the same reasons we have images for some repos that aren't even deployed in containers: I don't need to even install `node` (or `ruby`, or whatever) locally, and operations on the repo should pretty much only ever depend on `docker`.

> oh, I might see my mistake -- is it running those 3 docker commands in 3 separate containers or something like that? 😞

Yep. 

Since you want to modify more than one file, I think what you really want to do is to mount in `$PWD` to `/usr/src/app`.

You could also just make a `bin/yarn` stub in this repo to run yarn with given args with the correct mounting, rather than making a make task for every possible command. We do this in several other repos for related tools.